specification/vmware/resource-manager/readme.md

```diff
@@ -21,7 +21,38 @@ These are the global settings for the VMware Solution API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-03-20
+tag: package-2020-07-17-preview
+```
+
+### Tag: package-2020-07-17-preview
+
+These settings apply only when `--tag=package-2020-07-17-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-07-17-preview'
+input-file:
+- Microsoft.AVS/preview/2020-07-17-preview/vmware.json
+directive:
+  - suppress: R3020
+    from: vmware.json
```

Comment:
could you specify the  swagger you want to suppress, There may be several versions of "vmware.json"

Reply:
Hi @jianyexi. I don't understand. Can you provide an example?

Reply:
E.g., 'from: vmware.json'  --> 'from:Microsoft.AVS/preview/2020-07-17-preview/vmware.json'

Reply:
We actually want these suppressions in every vmware.json version.